packages/manager/modules/web-paas/src/add/add.controller.js

```diff
@@ -126,10 +139,7 @@ export default class WebPassAddCtrl {
 
   scrollToTop() {
     this.$timeout(() => {
-      document.getElementById('web-pass-add-header').scrollIntoView({
-        behavior: 'smooth',
-      });
-      document.getElementById('web-pass-add-alert').focus();
+      document.getElementById(this.PAGE_SECTION_HEADER).scrollIntoView(false);
```

Comment:
use $anchorScroll
scrollToTop() {
this.$location.hash(this.PAGE_SECTIONS.BACK_BUTTON);
this.$anchorScroll();
}

Reply:
Tried `AnchorScroll` but faced the same issue. This resolved the issue